crates/stackable-operator/src/kvp/mod.rs

```diff
@@ -401,28 +242,15 @@ mod test {
 
     #[test]
     fn try_from_tuple() {
-        let label = Label::try_from(("stackable.tech/vendor", "Stackable")).unwrap();
+        let label =
+            KeyValuePair::<LabelValue>::try_from(("stackable.tech/vendor", "Stackable")).unwrap();
```

Comment:
note: I guess we should use the `Label` type alias here?

```suggestion
        let label = Label::try_from(("stackable.tech/vendor", "Stackable")).unwrap();
```

Reply:
8f6daa0333b01541fa428d6a98dd49ad3bc85d33